luigi/parameter.py

```diff
@@ -44,6 +44,12 @@
 _no_value = object()
 
 
+class ParameterVisibility(object):
```

Comment:
You can use the enum class that's built into python. (there's a 2.7 backport we're already using)